bionic/flow.py

```diff
@@ -1271,7 +1335,25 @@ def __dir__(self):
     def __getattr__(self, name):
         def partial(*args, **kwargs):
             return self._wrapped_method(name, *args, **kwargs)
-        partial.__doc__ = self._flow.entity_docstring(name)
+
+        partial.__name__ = name
+
+        doc_prefix =\
+            self._docstring_prefix_template.format(name=name)
+        entity_docstring = self._flow.entity_docstring(name)
+        if entity_docstring is None:
+            main_docstring = doc_prefix + "."
+        else:
+            clean_entity_docstring = dedent(entity_docstring).strip()
+            main_docstring = f"{doc_prefix}:\n{clean_entity_docstring}"
+        method_name = self._wrapped_method.__name__
+
+        partial.__doc__ = (
```

Comment:
The template looks great! 

nit: I confess it is hard for me to understand what the intention is just reading this code, so it might not hurt to make that extremely obvious to mere mortals with a test for the `help` output like https://github.com/square/bionic/blob/master/tests/test_flow/test_api.py#L652

Reply:
Thanks, great idea! I'll do that.

Reply:
I added slightly more detailed tests and some comments here. Did that help?

Reply:
"slightly more" is an understatement :grin: